src/Item_Ticket.php

```diff
@@ -582,7 +582,7 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
      *
      * @return void
      **/
-    public static function dropdownMyDevices($userID = 0, $entity_restrict = -1, $itemtype = 0, $items_id = 0, $options = [])
+    public static function dropdownMyDevices($userID = 0, $entity_restrict = -1, $itemtype = '0', $items_id = 0, $options = [])
```

Comment:
I guess an empty string is more logical. Anyway, this parameter is useless (but we cannot remove it in the `10.0/bugfixes` branch) as it is overriden in every foreach loop that is using the `$itemtype` variable value.

```suggestion
    public static function dropdownMyDevices($userID = 0, $entity_restrict = -1, $itemtype = '', $items_id = 0, $options = [])
```